functions/slack/main.py

```diff
@@ -14,21 +14,16 @@
 # [START functions_slack_setup]
 import hashlib
 import hmac
-import json
 import os
 
-import googleapiclient
+import googleapiclient.discovery
 from flask import jsonify
 
-with open('config.json', 'r') as f:
-    data = f.read()
-config = json.loads(data)
-
 
 kgsearch = googleapiclient.discovery.build(
     'kgsearch',
     'v1',
-    developerKey=os.environ.get('API_KEY') or config.get('KG_API_KEY'),
+    developerKey=os.environ['KG_API_KEY'],
```

Comment:
This will result in a `KeyError` if this isn't in the environment. 

Is passing a API key the recommended way to use this API? If there are credentials in the environment you can do `googleapiclient.discovery.build(...)` and the library will use ADC

Reply:
This API _does_ require an API key. (I got an error when removing the `developerKey` parameter.)